pkg/util/rand/rand_test.go

```diff
@@ -84,3 +84,23 @@ func TestShuffle(t *testing.T) {
 		t.Errorf("Shuffle(%v) => %v, want %v", have, got, want)
 	}
 }
+
+func TestIntnRange(t *testing.T) {
+	// 0 is invalid.
+	for min, max := range map[int]int{1: 2, 10: 123, 100: 500} {
+		inrange := IntnRange(min, max)
+		if inrange < min || inrange > max {
```

Comment:
Apologies for the drive-by review, but since I'm build cop and looking for safe things to merge...

Generating a single number per test here is probably not sufficient, could take a long time to expose a problem. Suggest doing this 100+ times per test.

Second, these functions both have a bug/undocumented feature, which is that they'll never return the top number in the range. Either document or fix.